### ISSUE TITLE
AndroidManifest.xml - error

### ISSUE THREAD

**Issue by lovkyndig** [GoogleChromeLabs/bubblewrap#784]
## `bubblewrap build` give error about missing AndroidManifest.xml
![image](https://user-images.githubusercontent.com/125480548/233017851-f0754cb0-20d2-459e-bd10-b4e09b1d6cd4.png)

## What is the right path to the tool who create _AndroidManifest.xml_?
I am using windows and have to set the path manually.
The easy part is to set the path to `jdk-11.0.9.1+1/bin`.
But the `android_jdk`-directory have many subdirectories, and I have set the path to:
- C:\Users\my-account\.bubblewrap\android_sdk\build-tools\33.0.2
- C:\Users\my-account\.bubblewrap\android_sdk\tools
- C:\Users\my-account\.bubblewrap\android_sdk\tools\bin

## Is it something wrong with my paths?
I can see the build-command is creating the directories, but there is no file in the directory:
app/build/intermediates/merged_manifest/release/out/

**Comment by lovkyndig** [comment]
It is the latest version, up to date.

**Comment by andreban** [comment]
Would you mind running and sharing the output of `bubblewrap doctor`?

**Comment by lovkyndig** [comment]
![image](https://user-images.githubusercontent.com/125480548/233026201-07bf1102-a2cd-411a-ae4e-b0cc43f312dc.png)

**Comment by andreban** [comment]
You JDK and Android SDK seem to be fine (as highlighted by doctor). At this point, Bubblewrap is a thing wrapper over Gradle, [invoking `gradlew.bat bundleRelease`](https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/cli/src/lib/cmds/build.ts#L123). If you have your JDK and Android SDK paths set in the enviroment, would you mind trying to run `gradlew.bat bundleRelease` directly? I suspect will get a similar error and can move to find what's going on in that bit.

**Comment by lovkyndig** [comment]
## bash: gradlew: command not found
![image](https://user-images.githubusercontent.com/125480548/233040159-c72efd31-3af8-494f-ae96-6e2c036a42e5.png)

**Comment by lovkyndig** [comment]
## gradlew bundleRelease --scan give this msg:
![image](https://user-images.githubusercontent.com/125480548/233047439-60fb8ec2-3ae9-4ac8-a44a-5240dc2d3f0c.png)

**Comment by andreban** [comment]
Looks like Gradle is trying to download something, but failing - not sure if that's related to the previous one, though.

**Comment by andreban** [comment]
> ## bash: gradlew: command not found
> ![image](https://user-images.githubusercontent.com/125480548/233040159-c72efd31-3af8-494f-ae96-6e2c036a42e5.png)
> 
> This message is when I'm not using windows-cmd, but `git-bash`.
> 
> ## When using windows-cmd:
> ![image](https://user-images.githubusercontent.com/125480548/233042992-5db3b6a0-bb6f-4699-8541-f3d04c81839b.png)
> 
> ## After setting JAVA_HOME and ANDROID_SDK_ROOT to the root - this message:
> ![image](https://user-images.githubusercontent.com/125480548/233045387-66669a04-f6a6-4a70-b3ef-eaf74ea0385d.png)

Ah, this one reproduces it - so, yeah, it's definitely a Gradle issue. Let me see if I can find something on what causes it.

**Comment by lovkyndig** [comment]
Here is some reports from `gradlew bundleRelease`:
- [stacktrace.txt](https://github.com/lovkyndig/stacktrace.txt)
- [debug.txt](https://github.com/lovkyndig/debug.txt)
- [info.txt](https://github.com/lovkyndig/info.txt)

**Comment by andreban** [comment]
Doing some digging, this seems similar to the issue reported on Flutter at https://github.com/flutter/flutter/issues/76668. It seems related to two things:

- The target application not being on drive C:
- The filesystem not being NTFS

I'll try to replicate on a Windows computer - if that's the case, unfortunately this seems more like an issue with the Android SDK / Gradle, and there may be little that can be done on the Bubblewrap side (I'll investigate though, but wanted to set expectations).

**Comment by lovkyndig** [comment]
Thank you so much! I'm very impressed (and glad) of the will to help and the guide to find out of the problem. After your research I'll try to remove my local repo from my SD-drive (exFAT) to C (NTFS). Results coming soon!

**Comment by andreban** [comment]
I managed to reproduce on a Windows machine, on drive E, with a FAT32 pendrive connected.

Moving to an NTFS filesystem fixes it. I initially got the same issue you mentioned above, but solved it by deleting the `.gradle` and `app/build` directories, to get rid of any trace left of previous build attempts. I also changed `twa-manifest.json` to make `signingKey.path` point to the new, correct location.
